src/Tests/FlatSharpCompilerTests/FieldNameNormalizationTests.cs

```diff
@@ -19,7 +19,7 @@ namespace FlatSharpTests.Compiler;
 public class FieldNameNormalizationTests
 {
     [Fact]
-    public void NullableAnnotations()
+    public void NormalizeFieldNames()
```

Comment:
Note, I renamed this as it seems like it was a copy/paste issue from `NullableAnnotationTests`

Reply:
That sounds like something I'd do. Thanks for catching it!